### DISCUSSION TITLE
ASReview 0.19 RIS export error: "float object is not iterable"

### DISCUSSION THREAD

**Discussion by jaohls** [asreview/asreview#988] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
Hi there all you excellent ASReview people,

Stumbled upon this problem - a colleague in my team cannot export to RIS file after screening ~25 % of the data.
![image](https://user-images.githubusercontent.com/96236396/155088009-363610db-f3d7-4252-a0dc-9e9088b1ef1b.png)

Export to CSV and Excel formats seem to work.

Uploaded the .asreview project file in question here: https://drive.protonmail.com/urls/NTB1H2V6QG#tQ68AB753Apx

**Comment by ottomattas** [comment, marked as answer]
Hello @jaohls,
Thanks for starting this discussion. Can you tell me more about your initial dataset or maybe even share it? What format was it in?

**Reply by jaohls** [reply]
Hi, Search results as an EndNote library, exported in RIS format to Rayyan for deduplication then imported to ASR (exactly what format was exported from Rayyan is unsure). I'll try to get a hold of the exact file that was imported to ASReview. The initial EN library was this one:  https://drive.protonmail.com/urls/MSV9846KE4#8B3RDnf6sXtM

**Reply by J535D165** [reply]
Hi @jaohls, thanks for reporting. This is an issue with ASReview. We are working on a solution and we will release version 0.19.1 as soon as possible. After upgrading, everything should work. 

Progress https://github.com/asreview/asreview/issues/992

**Reply by FelixWdm** [reply]
Hi @jaohls , thank you for sharing the issue and including the EndNote library. While the technical aspects are being adresses, I have was wondering why you chose Rayyan to deduplicate this set. Our experience is that EndNote is more reliable in the deduplication. Using your library, I used EndNote to exlude another 5 duplicate records. Rayyan also removes the DOI from the records, which limits the functionality of ASReview as well, we use it to offer a link to the full text.

**Reply by J535D165** [reply]
Hi @jaohls, there is a new release of ASReview (0.19.1). Please upgrade to see the fix.

**Reply by jaohls** [reply]
Hi @J535D165 and ASReview team, thanks a bunch for the swift work  !!

**Reply by jaohls** [reply]
Hi @FelixWdm, thanks for the comment! For context, we have several datasets on different subjects in our review that we need to process in the same way for consistency. Even after EndNote deduplication by our research librarian there were quite a number of duplicates, and we somewhat arbitrarily used Rayyan to process those. Might not have been the best solution regarding to the DOI as you stated, but managed that by using the update records function in endnote afterwards....

Edit: spelling

**Reply by FelixWdm** [reply]
Hi @jaohls , thanks for the update. I do understand the use case now, unfortunately not all is fully interoperable yet, but glad you brought this up, so we could fix it on our end.
